ci/cpu/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+# Copyright (c) 2018-2022, NVIDIA CORPORATION.
+##############################################
+# kvikIO CPU conda build script for CI       #
+##############################################
+set -e
+
+# Set path and build parallel level
+export PATH=/opt/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=${PARALLEL_LEVEL:-4}
+
+# Set home to the job's workspace
+export HOME="$WORKSPACE"
+
+# Determine CUDA release version
+export CUDA_REL=${CUDA_VERSION%.*}
+
+# Setup 'gpuci_conda_retry' for build retries (results in 2 total attempts)
+export GPUCI_CONDA_RETRY_MAX=1
+export GPUCI_CONDA_RETRY_SLEEP=30
+
+# Use Ninja to build, setup Conda Build Dir
+export CMAKE_GENERATOR="Ninja"
+export CONDA_BLD_DIR="$WORKSPACE/.conda-bld"
+
+# Switch to project root; also root of repo checkout
+cd "$WORKSPACE"
+
+# If nightly build, append current YYMMDD to version
+if [[ "$BUILD_MODE" = "branch" && "$SOURCE_BRANCH" = branch-* ]] ; then
+  export VERSION_SUFFIX=`date +%y%m%d`
+fi
```

Comment:
@madsbk do we intend to do releases on RAPIDS' release cycle, or more/less frequently? Depending on that answer, we might want to either

- rename the main dev branch of this repo to `branch-22.04` and leave these lines
- keep the main branch as is and remove these lines along with some below

Reply:
Yes, I think we should follow the RAPIDS' release cycle in addition to the nightly releases.